### ISSUE TITLE
3D Swipe

### ISSUE THREAD

**Issue by AlirezaAzizi145** [archriss/react-native-snap-carousel#561]
Hello guys. how can i create Something Like [this ](https://github.com/Bobby-hewitt/react-native-3d-swiper#readme)in react-native-snap-carousel?

i think if we can show both of next and previous cards in "Stack  of Cards" layout we can create  it but i dont know how

**Comment by tonylibra** [comment]
I have the same problem. I did a quick experiment on the example project in the repo, and I found that `zIndex` cannot be an animation property.

**Comment by AlirezaAzizi145** [comment]
![snap_carousel](https://user-images.githubusercontent.com/30459434/62423136-8dce3680-b6d2-11e9-9a90-7a6481440cb9.jpg)
i need to move this card behind the active card like left side

**Comment by tonylibra** [comment]
This is what I have, because of the zIndex cannot be animated
![Screen Shot 2019-08-05 at 09 15 30](https://user-images.githubusercontent.com/7263838/62432353-ca953e80-b761-11e9-9455-9dde861bd277.png)

**Comment by AlirezaAzizi145** [comment]
Can you share code of this style?

You only change animation.js?

If you can please share this for me I want to show someone maybe he can help us

**Comment by AlirezaAzizi145** [comment]
@bd-arc  any solution?

**Comment by AlirezaAzizi145** [comment]
@tonylibra  do you play with elevation?

**Comment by bd-arc** [comment]
@AlirezaAzizi145 I've written extensively about it in [the dedicated doc](https://github.com/archriss/react-native-snap-carousel/blob/master/doc/CUSTOM_INTERPOLATIONS.md#caveats).

I encourage you to read everything related to the caveats since some of the workaround suggestions might fit your use case.

**Comment by AlirezaAzizi145** [comment]
@bd-arc yes. i played with elevation instead z-index but still i have a problem

**Comment by AlirezaAzizi145** [comment]
@bd-arc no solution? no sample?

**Comment by AlirezaAzizi145** [comment]
@tonylibra Can You Find Any Solution? I Really Need This Carousel Style

**Comment by bd-arc** [comment]
@AlirezaAzizi145 You will probably need to rely on a custom `PanResponder` for that instead of on this plugin since the `zIndex` property can't be animated — RN limitation.

**Comment by AlirezaAzizi145** [comment]
@tonylibra if you find any solution i will be grateful to send me the soulution

**Comment by sebastijandumancic** [comment]
This is shown like this in the example gif. Is there a prop to do this? Thanks!

**Comment by bd-arc** [comment]
@sebastijandumancic I don't recall any gif displaying one card on top of the ones before as well as the ones after.

Can you please share the link of the image you're referring to?

**Comment by sebastijandumancic** [comment]
Sorry, I was thinking about standard horizontal scroll which shows a bit of previous and next slide. I've achieved it using item width prop.

**Comment by naveedulaziz** [comment]
If anybody find any solution please share it with me. I also have to make something similar

**Comment by wuhuanda** [comment]
> I have the same problem. I did a quick experiment on the example project in the repo, and I found that `zIndex` cannot be an animation property.

Do you have any experienced code or demo?

**Comment by dohooo** [comment]
Sorry, please allow me to advertise for my open source library! ~
I think this library [react-native-reanimated-carousel](https://github.com/zwh1666258377/react-native-reanimated-carousel) will solve your problem. It is a high performance and very simple component, complete with React-Native reanimated 2
